presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergTableHandle.java

```diff
@@ -78,14 +72,9 @@ public Optional<String> getTableSchemaJson()
         return tableSchemaJson;
     }
 
-    public SchemaTableName getSchemaTableName()
-    {
-        return new SchemaTableName(schemaName, tableName.getTableName());
-    }
-
     public SchemaTableName getSchemaTableNameWithType()
     {
-        return new SchemaTableName(schemaName, tableName.getTableNameWithType());
+        return new SchemaTableName(getSchemaName(), icebergTableName.getTableNameWithType());
```

Comment:
Where is this function being used? If no use, please remove it

Reply:
Done